pkg/stub/catalogsourceconfig_handler.go

```diff
@@ -0,0 +1,77 @@
+package stub
+
+import (
+	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
```

Comment:
I am trying to keep the code succinct and hence leaving out the `_v1alpha` bit. I hope that is okay as `olm` is indicator enough that we are dealing with an OLM item.

Reply:
I think, it would be better to keep it this way because the subsequent parts of the code should be oblivious to changes we are making here and it would be less hassle when moving from one version to other.